eland/operations.py

```diff
@@ -800,6 +800,34 @@ def quantile(
         else:
             return df if is_dataframe else df.transpose().iloc[0]
 
+    def unique(self, query_compiler: "QueryCompiler") -> pd.Series:
+
+        query_params, _ = self._resolve_tasks(query_compiler)
+
+        fields = query_compiler._mappings.all_source_fields()
+
+        body = Query(query_params.query)
+
+        # Unique is only for eland.Series
+        assert len(fields) == 1
+
+        body.composite_agg_bucket_terms(
+            name=f"unique_{fields[0].column}",
```

Comment:
Since `f"unique_{fields[0].column}"` is calculated multiple times we should use a variable.